api/proto/teleport/legacy/client/proto/authservice.proto

```diff
@@ -507,6 +515,18 @@ message Features {
   // Policy enables the Teleport Policy feature set.
   // At the time of writing, this includes Teleport Access Graph (TAG).
   PolicyFeature Policy = 28 [(gogoproto.jsontag) = "policy,omitempty"];
+  // Questionnaire indicates whether cluster users should get an onboarding questionnaire
+  bool Questionnaire = 29;
+  // IsStripeManaged indicates if the cluster billing is managed via Stripe
+  bool IsStripeManaged = 30;
+  // ExternalAuditStorage indicates whether the EAS feature is enabled in the cluster.
+  bool ExternalAuditStorage = 31;
+  // SupportType indicates the type of the customer's support
+  SupportType SupportType = 32;
+  // JoinActiveSessions indicates whether joining active sessions via web UI is enabled
+  bool JoinActiveSessions = 33;
+  // MobileDeviceManagement indicates whether endpoint management (like Jamf Plugin) can be used in the cluster
+  bool MobileDeviceManagement = 34;
```

Comment:
I think we should be consistent with other fields on the json tag format (snake vs camel). With current impl, the features object will be mixed with attributes like `product_type` and `MobileDeviceManagement`

Reply:
Thanks, fixed in  https://github.com/gravitational/teleport/pull/39695/commits/c8e7f6324d02263d254bbb55ba2dc59c1ceab72e

Reply:
I think it should be `mobile_device_management` instead of `mobileDeviceManagement`